test/test_maven_registry.d

```diff
@@ -0,0 +1,19 @@
+/+dub.sdl:
+dependency "vibe-d" version="~>0.8.3-alpha.1"
```

Comment:
0.8.3 has been released for a while now.

Reply:
By changing to 0.8.3 there are some failing tests:
Root package test_maven_registry reference vibe-d ~>0.8.3 cannot be satisfied.
https://travis-ci.org/dlang/dub/jobs/357730493